docs/configuration.md

```diff
@@ -371,15 +371,18 @@ The possible values are: `gtc` (default), `fok` or `ioc`.
 Freqtrade is based on [CCXT library](https://github.com/ccxt/ccxt) that supports over 100 cryptocurrency
 exchange markets and trading APIs. The complete up-to-date list can be found in the
 [CCXT repo homepage](https://github.com/ccxt/ccxt/tree/master/python). However, the bot was tested
```

Comment:
"the bot was tested" -- this should be more clear, I guess.
Maybe "the bot was tested by developers", or "the bot was officially tested by the development team"